sql-statements/sql-statement-admin-evolve-bindings.md

```diff
@@ -0,0 +1,203 @@
+
+..
+
++------------------------------+----------------------------------------------------------------------------------------+------------+----------------+-------------------------+-------------------------+---------+-----------------+---------+
+| Original_sql                 | Bind_sql                                                                               | Default_db | Status         | Create_time             | Update_time             | Charset | Collation       | Source  |
++------------------------------+----------------------------------------------------------------------------------------+------------+----------------+-------------------------+-------------------------+---------+-----------------+---------+
+| select c from t1 where b = ? | SELECT /*+ use_index(@`sel_1` `test`.`t1` `b`)*/ `c` FROM `t1` WHERE `b`=1234          | test       | using          | 2020-09-29 21:30:33.978 | 2020-09-29 21:30:33.978 |         |                 | capture |
+| select c from t1 where b = ? | SELECT /*+ use_index(@`sel_1` `test`.`t1` `b_2`)*/ `c` FROM `test`.`t1` WHERE `b`=1234 | test       | pending verify | 2020-09-29 21:32:20.978 | 2020-09-29 21:32:20.978 | utf8    | utf8_general_ci | evolve  |
++------------------------------+----------------------------------------------------------------------------------------+------------+----------------+-------------------------+-------------------------+---------+-----------------+---------+
+2 rows in set (0.00 sec)
+
++------------------------------+----------------------------------------------------------------------------------------+------------+----------+-------------------------+-------------------------+---------+-----------------+---------+
+| Original_sql                 | Bind_sql                                                                               | Default_db | Status   | Create_time             | Update_time             | Charset | Collation       | Source  |
++------------------------------+----------------------------------------------------------------------------------------+------------+----------+-------------------------+-------------------------+---------+-----------------+---------+
+| select c from t1 where b = ? | SELECT /*+ use_index(@`sel_1` `test`.`t1` `b`)*/ `c` FROM `t1` WHERE `b`=1234          | test       | using    | 2020-09-29 21:30:33.978 | 2020-09-29 21:30:33.978 |         |                 | capture |
+| select c from t1 where b = ? | SELECT /*+ use_index(@`sel_1` `test`.`t1` `b_2`)*/ `c` FROM `test`.`t1` WHERE `b`=1234 | test       | rejected | 2020-09-29 21:32:20.978 | 2020-09-29 21:32:23.978 | utf8    | utf8_general_ci | evolve  |
++------------------------------+----------------------------------------------------------------------------------------+------------+----------+-------------------------+-------------------------+---------+-----------------+---------+
+2 rows in set (0.00 sec)
+```
+
+TODO: Why is the plan above rejected????
```

Comment:
A evolved plan is marked as accepted only if it runs 1.5 times faster than the existing bound plan, have you run the evolved plan manually to check the execution time?

Reply:
I have not. I did not know this 1.5x requirement, I will update the examples to show both rejected and accepted examples.